Jint/Runtime/Interpreter/Statements/JintStatement.cs

```diff
@@ -32,8 +32,11 @@ protected JintStatement(Engine engine, Statement statement)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public Completion Execute()
         {
-            _engine._lastSyntaxNode = _statement;
-            _engine.RunBeforeExecuteStatementChecks(_statement);
+            if (_statement.Type != Nodes.BlockStatement)
```

Comment:
We might need more options later, like being able to define what statements should be ignored, or if Blocks should step or not. Would even be fine to just say that a block is stepped in and change the test.